wca-regulations.md

```diff
@@ -776,3 +803,58 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
+        - I3a2) For subsequent stages, X must be the same or higher than the previous stage.
+    - I3b) Each set comprises a number of points. The winner of a set is the first competitor to win 3 points.
+    - I3c) The winner of a point is the competitor with the better result, where "better" is defined as the shorter time. A DNF or DNS is the worst possible result.
+    - I3d) If neither competitor has a better result, neither wins the point.
+    - I3e) If neither competitor wins a set after 7 points the competitor with the most points wins the set. If both competitors have the same number of points, the competitor with the best single result in the set wins the set. If neither competitor has a best single result in the set, the best single results in the match and then single results in the round are considered. If necessary, the competitors' seeds act as a final tiebreaker, with the higher-seeded competitor winning the set.
+- I4) Head to Head speed solving procedures:
+    - I4a) Standard speed solving procedures are followed, as described in [Article A](regulations:article:I). Additional regulations that supersede the corresponding procedures in Article A are described below.
+    - I4b) Scrambling:
+        - I4b1) Competitors in a match must use the same scramble sequences as each other for a point. Each point must use different scramble sequences and each match must use different scramble sequences.
+    - I4c) Inspection:
+        - I4c1) When both competitors are present at the solving station and when each judge is ready, the announcer asks "READY?".
+        - I4c2) When both competitors have confirmed they are ready, or once at least 15 seconds have passed, the announcer calls “START” and starts timing the inspection, beginning the point.
+        - I4c3) When 8 seconds of inspection have elapsed, the announcer calls "8 SECONDS".
+        - I4c4) When 11 seconds, 12 seconds, and 13 seconds of inspection have elapsed, the announcer calls “3", “2”, and “1” respectively.
+        - I4c5) When 14 seconds of inspection have elapsed, the announcer calls “GO”. The competitor must start the solve (see [Regulation A4d](regulations:regulation:A4d)) at this point. Penalty for intentionally delaying the start of the solve: disqualification of the attempt (DNF) at the discretion of the judge.
+        - I4c6) Due to modified inspection procedures, the usual penalties for not starting within 15 seconds (see [Regulation A4d1](regulations:regulation:A4d1) and [Regulation A4d2](regulations:regulation:A4d2)) do not apply in Head to Head rounds.
+        - I4c7) The competitor must not start the solve before the announcer calls “GO”. Penalty: disqualification of the attempt (DNF) at the discretion of the judge.
+    - I4d) Recording results:
+        - I4d1) The judge notes whether a competitor won or lost a point, or there was no winner, on the score sheet when recording the result.
```

Comment:
```suggestion
        - I4d1) When recording the result, the judge also notes whether a competitor won or lost a point, or there was no winner, on the score sheet.
```

Reply:
Clarification for feedback received here: https://forum.worldcubeassociation.org/t/january-2026-regulations-feedback-introduce-head-to-head-format/65782/44

Recording the result (A7b) already requires writing the time down because the result is measured based on the time (9f1), but adding "also" stresses that this is an addition, not a replacement.